docs/introduction-bdd.md

```diff
@@ -0,0 +1,97 @@
+
+## Behavior Driven Development(BDD):
+
+   Behavior driven development (or BDD) is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.
```

Comment:
since we are in agile development i add agile word there.. not a big deal if we delete "agile" word